internal/cli/generate.go

```diff
@@ -64,5 +69,8 @@ func init() {
 
 	generateCmd.Flags().BoolVar(&emitSQLSupport, "sql", false, "Emit code for SQL support")
 
+	generateCmd.Flags().BoolVar(&limitPathLength, "limitlength", false, "Limit path length to the number of character passed to maxlength")
+	generateCmd.Flags().IntVar(&maxPathLength, "maxlength", generator.DefaultMaxPathLength, "Limit path length to the given number of characters")
```

Comment:
I would prefer to have a single flag to control this. For example something like:

```suggestion
	generateCmd.Flags().IntVar(&maxPathLength, "max-path-length", generator.DefaultMaxPathLength, "Maximum length of filename. Set to 0 for no limit")
```